test/ETO/ETOCommitment.js

```diff
@@ -1119,10 +1178,9 @@ contract("ETOCommitment", ([deployer, admin, company, nominee, ...investors]) =>
     it("should enable transfer on equity token on success", async () => {
       await deployETO({
         ovrETOTerms: {
-          MIN_TICKET_EUR_ULPS: constETOTerms.MIN_QUALIFIED_INVESTOR_TICKET_EUR_ULPS,
+          MIN_TICKET_EUR_ULPS: Q18.mul(100),
           MAX_TICKET_EUR_ULPS: Q18.mul(15000000),
           ENABLE_TRANSFERS_ON_SUCCESS: true,
-          ALLOW_RETAIL_INVESTORS: false,
```

Comment:
where is ETOConstraints with transferable token explicitly passed? we should force such constraints for test to make sense

Reply:
I would use our presets in tests... that you declared in migrations. you can move this to tests or we can make copy

Reply:
There are default contraints terms, very similar to all the other contract deployments that there were before. I would actually only specifically add certain values if they are somehow expected in the test. It's actually done a few times in a number of scenarios if you search for: ovrETOTermsConstraints